src/common/constants.js

```diff
@@ -104,3 +104,4 @@ export const twitter_non_eu_url = 'https://twitter.com/derivdotcom/'
 export const twitter_uk_url = 'https://www.twitter.com/deriv_uk/'
 export const twitter_eu_url = 'https://www.twitter.com/deriv_eu/'
 export const cms_assets_end_point = 'https://cms.deriv.cloud/assets/'
+export const loss_percent = 62
```

Comment:
@prince-deriv @sean-binary Shouldn’t  we use capitalize names for constant variables ?
```ts
const LOSS_PERCENT = 62
```

Reply:
Yes we should but that can be handle in utility typescript refactor after we refactor all the pages first.